olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -237,6 +241,11 @@ set(OLP_SDK_PARSER_SOURCES
     ./src/client/parser/ApiParser.h
 )
 
+set(OLP_SDK_REPOSITORY_SOURCES
+    ./src/client/repository/ApiCacheRepository.cpp
+    ./src/client/repository/ApiCacheRepository.h
+)
```

Comment:
The logic here is a little broken. Why do you place `client/api/` files under `OLP_SDK_CLIENT_SOURCES` but `client/repository/` filer under `OLP_SDK_REPOSITORY_SOURCES`? Please put them all under the `OLP_SDK_CLIENT_SOURCES` as they are all under `client` folder.